camlibs/ptp2/library.c

```diff
@@ -3081,6 +3085,11 @@ camera_exit (Camera *camera, GPContext *context)
 		gp_port_usb_clear_halt
 				(camera->port, GP_PORT_USB_ENDPOINT_INT);
 	}
+#if defined(HAVE_LIBWS232) && defined(WIN23)
```

Comment:
WIN23 or WIN32? (typo?)

Reply:
That was indeed a typo - it is now fixed.